### ISSUE TITLE
How to grab video metadata

### ISSUE THREAD

**Issue by nfunwigabga** [videojs/videojs-youtube#450]
Hi.
Normally videojs provides a method to grab video metadata such as `player.duration()`. This works for regular mp4 videos...is it possible to get this information for YouTube videos (and for Vimeo for the videojs-vimeo plugin)?

**Comment by bencergazda** [comment]
#414

**Comment by dinony** [comment]
Yep, this is not working: I've also added an example [here](https://github.com/dinony/videojs-youtube-issue0).
